wallet.md

```diff
@@ -54,41 +50,36 @@ Click the `ALL TRANSACTIONS` button to access your wallet transactions log scree
 Follow [this guide](wallet_logs.md) to learn how to work with the transactions log screen.
 
 ## Sending Smesh
-Follow the steps in [this guide](send_coin.md) to send Smesh to any other account.
+Follow the steps in [this guide](send_coin.md) to send SMH to another account.
```

Comment:
Smesh is more general. You actually currently send Smidge...

Reply:
This terminology is very confusing. "Smesh" and "SMH" should be the same. It's bad enough that "Smesh" also refers to "Smeshing." I'll replace with something more generic like "coin."

(For the record, in Ethereum, it's called "sending ETH" even if I'm only sending a few wei.)

Reply:
So let's keep send SMH for now. We'll revisit coin names for sm 0.2 as planned.